### PR TITLE
move case functions and char predicates back to Base

### PR DESCRIPTION
**Pull request by JeffBezanson**
Here's my proposed resolution to #25394.

### CODE REVIEW

**Comment by nalimilan** [comment]
If we do that, better also move back the three remaining functions to Base. I don't really see the point of having a module only for those, given that we would have Unicode-dependent functions in Base.

**Comment by JeffBezanson** [comment]
They're unicode-dependent, but not unicode-specific. And Base itself calls these functions in many places, so it would not really be de-coupled anyway.

File: base/exports.jl [hunk 1]
```diff
@@ -704,6 +704,19 @@ export
     hex2bytes!,
     info,
     isascii,
+    isalpha,
+    isascii,
```
Comment (StefanKarpinski) [inline base/exports.jl]: `isascii` is duplicated.

**Review by StefanKarpinski — APPROVED** [review]

**Comment by StefanKarpinski** [comment]
I'm fine with this, let's do it.

File: stdlib/Unicode/src/Unicode.jl [hunk 2]
```diff
@@ -4,19 +4,12 @@ __precompile__(true)
 
 module Unicode
 
-using Base.Unicode: normalize, graphemes, isassigned, textwidth, isvalid,
-                    islower, isupper, isalpha, isdigit, isxdigit, isnumeric, isalnum,
-                    iscntrl, ispunct, isspace, isprint, isgraph,
-                    lowercase, uppercase, titlecase, lcfirst, ucfirst, iscased
+using Base.Unicode: normalize, graphemes, isassigned, iscased
```
Comment (nalimilan) [inline stdlib/Unicode/src/Unicode.jl]: Why have you kept `iscased` in Unicode? It feels lonely now that all case functions have been moved back to Base.

**Pull request merged**